.cargo/config

```diff
@@ -0,0 +1,2 @@
+[alias]
+ex = "run -p examples --bin"
```

Comment:
It would be nice to update the README section about running the examples to mention the existence of this alias.